src/providers/ffz/FfzEmotes.cpp

```diff
@@ -192,32 +195,39 @@ void FfzEmotes::loadChannel(
 
         .timeout(20000)
         .onSuccess([emoteCallback = std::move(emoteCallback),
-                    modBadgeCallback =
-                        std::move(modBadgeCallback)](auto result) -> Outcome {
+                    modBadgeCallback = std::move(modBadgeCallback),
+                    &channel](auto result) -> Outcome {
             auto json = result.parseJson();
             auto emoteMap = parseChannelEmotes(json);
             auto modBadge = parseModBadge(json);
 
             emoteCallback(std::move(emoteMap));
             modBadgeCallback(std::move(modBadge));
+            channel.addMessage(makeSystemMessage("FFZ: emotes reloaded."));
```

Comment:
For any public-facing messages, I think we should stick to the preferred branding of the service. Here it's "FrankerFaceZ". and making the messages just a proper sentence (e.g. "FrankerFaceZ channel emotes reloaded.") feels better to me.

Reply:
Fixed.